docs/source/developer_guide/get_started.rst

```diff
@@ -143,6 +146,19 @@ To document a new module, a new block must be added. For example, for the ``janu
         make clean; make html
 
 
+Notebook tutorials
+++++++++++++++++++
+
+Jupyter notebooks in ``docs/source/tutorials`` are automatically run by ``Sphinx`` using the
+`nbsphinx <https://nbsphinx.readthedocs.io/en/0.2.15/index.html>`_ extension, creating the :doc:`Tutorials </tutorials/index>`.
+
+
+These are tested before ``janus-core`` is published to PyPI, but can be tested locally by running::
+
+        cd docs
+        make clean; make tutorials
```

Comment:
Given the `all` default is `tutorials`, could do just `make`.

Reply:
Maybe better to be explicit though? Arguably was the same before with` make html`